test/common.js

```diff
@@ -453,3 +453,9 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
+  var startTime = new Date().getTime();
+  var stopTime =  startTime + time;
+  while (new Date().getTime() < stopTime);
```

Comment:
Hmm, in the new patch joyent/node will be using `Timer.now()` -- what's the difference between it and `Date.now()`?

https://github.com/joyent/node/pull/25763/files#diff-8736c5cbff21e1dee18b0c86d3d2689dR226